.github/workflows/website.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         with:
           node-version: 16
       - uses: bahmutov/npm-install@v1
-      - run: npm test --workspace packages/website
+      - run: npm run lint:ts -w packages/website
```

Comment:
nit: It's customary round these packages to use the CI Test step to run `npm test` and we configure the `test` run script in each project's package.json to run all the targets that should be run when testing the software, which can include linting.

using `npm-run-all` in the package.json run script definitions is our friend here, as it lets us have specific targets like `lint:ts` and then have the `test` target run all the other targets that make sense like `npm-run-all lint:ts test:widgets coverage etc`